docs: add dedicated package for code snippets from contracts section (#895)
FuelLabs/fuels-ts: apps/docs/src/guide/contracts/logs.md

# Working with Contract Logs

When you log a value within a contract method, it generates a log entry that is added to the log receipt, and the variable type is recorded in the contract's ABI. The SDK enables you to parse these values into TypeScript types.

Consider the following example contract:

<<< @/../../docs-snippets/contracts/log-values/src/main.sw#log-1{rust:line-numbers}

> To access the logged values in TypeScript, use the `logs` property found in the `FunctionInvocationResult` from a contract call result. The logs data will be stored in an `Array<any>`:

Reviewer: ```suggestion
To access the logged values in TypeScript, use the `logs` property in the `FunctionInvocationResult` of a contract call result. The logs data will be stored in an `Array<any>`:
```
